include/analytics.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/false
+
+# This is the server that is collecting zopen usage stats
+ZOPEN_STATS_URL="http://163.74.88.212:3000"
+
+isAnalyticsOn()
+{
+  jsonConfig="${ZOPEN_ROOTFS}/etc/zopen/config.json"
+  if [ ! -f ${jsonConfig} ]; then
+    printError "config.json file does not exist. This should not occur. Please report an issue"
+  fi
+  isCollecting=$(jq -r '.is_collecting_stats' $jsonConfig)
+  if [ "$isCollecting" = "true" ] || [ -z "${ZOPEN_ANALYTICS_JSON}" ]; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+isIBMHostname()
+{
+  ip_address=$(/bin/dig +short "$(hostname)" | tail -1)
+
+  if /bin/dig +short -x "${ip_address}" 2>/dev/null | grep -q "ibm.com"; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+sendStatsToRemote()
+{
+  json="$1"
+  response=$(curl -X POST -H "Content-Type: application/json" -d "$json" "${ZOPEN_STATS_URL}/statistics")
```

Comment:
how do you know curl is installed at this point?

Reply:
I'll add a check here